package.json

```diff
@@ -35,6 +35,7 @@
     "test:integration:base": "pnpm playwright test --config integration/playwright.config.ts",
     "test:integration:billing": "E2E_APP_ID=withBilling.* pnpm test:integration:base --grep @billing",
     "test:integration:cleanup": "pnpm playwright test --config integration/playwright.cleanup.config.ts",
+    "test:integration:custom": "pnpm test:integration:base --grep @custom",
```

Comment:
We can make these slightly faster by using a long running app for both sign and signup but lets merge this (and the stateProxy fix) and Ill handle it afterwards 👍🏻